docs/changelog-fragments/216.bugfix.2.rst

```diff
@@ -0,0 +1 @@
+Changed ``sftp.sftp_get`` to write files as bytes rather than assuming files are valid UTF8 -- by :user: `Qalthos`
```

Comment:
@Qalthos FYI this is a syntax error in RST :point_up: — there shouldn't be a space between the role `:user:` and its argument for Sphinx to parse that as one thing. Otherwise, it tries to interpret them separately.

This caused the failures here:
* https://github.com/ansible/pylibssh/pull/216/checks?check_run_id=2806860150
* https://readthedocs.org/projects/ansible-pylibssh/builds/13990866/

```suggestion
Changed ``sftp.sftp_get`` to write files as bytes rather than assuming files are valid UTF8 -- by :user:`Qalthos`
```